recipes/mdal-python/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
+if [ "$(uname)" == "Linux" ]; then
+   export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,${PREFIX}/lib"
+fi
+
+set CMAKE_GENERATOR=Ninja
```

Comment:
I guess this was intended to be `export`? Regardless you're not adding `ninja` as a build dependency so it's probably not needed.
```suggestion
```